csuntechlab/iSTART
# STR-143 -- FE: Create Embedded youtube video slide template

# Description
Created YouTube template based off mockup.

# Testing
1. Log in to `april.feldman`.
2. Click on alcohol module.
3. Check to see if it meets the mock-up criteria and responsiveness
4. Ensure that the continue button is not highlighted.
5. After 4-ish minutes, the video should be done and the continue button will be available.
6. Test for accessibility.


# Installation
Run `yarn install`; I added a vue youtube api package.
